rosbag2_storage_default_plugins/test/rosbag2_storage_default_plugins/sqlite/storage_test_fixture.hpp

```diff
@@ -91,7 +91,9 @@ class StorageTestFixture : public TemporaryDirectoryFixture
     std::unique_ptr<rosbag2_storage::storage_interfaces::ReadWriteInterface> writable_storage =
       std::make_unique<rosbag2_storage_plugins::SqliteStorage>();
 
-    writable_storage->open(temporary_dir_path_);
+    auto db_file = rosbag2_storage::FilesystemHelper::concat({temporary_dir_path_, "rosbag"});
```

Comment:
why that change? Why appending `rosbag` to the filename?

Reply:
`SqliteStorage` is being invoked directly here, so `db_file` needs to be the path to the actual database storage. Before this change, it was expecting the base folder since storage also interacted with the metadata file.

Reply:
also, looking at this again, I don't understand why `rosbag` is essential here.